src/ts/components/basic-components.ts

```diff
@@ -13,8 +19,8 @@ export const Button = styled.button`
   width: 311px;
   height: 45px;
   border-radius: 4px;
-  box-shadow: 0 3px 10px 0 rgba(72, 178, 228, 0.21);
-  background-color: #24b6e8;
+  box-shadow: 0 3px 10px 0 ${(p: SpeckleProps) => p.colorScheme.shadowColor};
```

Comment:
Shall we have some different styling for buttons when hover on, disabled?

Reply:
yeah, I think that'll be next step - polish the UIs